zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/IncidentHandlerTest.java

```diff
@@ -52,7 +52,7 @@ void testGetEntityType() {
   void shouldHandleRecord() {
     // given
     final Record<IncidentRecordValue> incidentRecord =
-        factory.generateRecord(ValueType.INCIDENT, r -> r.withIntent(ProcessIntent.CREATED));
```

Comment:
🤔 Wait why this didn't fail before?

Reply:
because `IncidentHandler.handlesRecord()` uses a string comparison not Intent enum comparison.
`return !intentStr.equals(IncidentIntent.RESOLVED.toString());`

I've changed this to compare enums on every handler I've moved, but not everyone does that.

🔧 @sdorokhova can we change that in handler?